configure.ac

```diff
@@ -1282,6 +1282,10 @@ AC_ARG_ENABLE([prof-libunwind],
 else
   enable_prof_libunwind="1"
 fi
+
+if test "x$enable_prof" = "x0" ; then
```

Comment:
This should go into the `enable_prof_libunwind="1"` branch? Right now it applies to both cases.

Reply:
You're right, I forgot about the `--enable-prof-libnwind=no` case. Thanks!